setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = iotp_orchestrator
-version = 1.6.0
+version = 1.7.0
```

Comment:
This version variable should be also adjusted in the release/1.7.0 branch (and the tag 1.7.0 moved to the new HEAD of release/1.7.0)?